docs/index.rst

```diff
@@ -3,50 +3,204 @@
+`releasing` the restraints of the unbound complex:
+
+.. math ::
+   \Delta G^\circ_\text{bind} = -(W_\text{attach} + W_\text{pull} + W_\text{release-conf} + W_\text{release-std})
+   :label: eq:bind
+
+
+.. image :: _static/images/pmf.png
+   :width: 600px
+   :align: center
+
+In the equation above, the work of releasing the restraints is split into two: :math:`W_\text{release-conf}` and
+:math:`W_\text{release-std}`. These represents the work for releasing conformational restraints (applied to host or
+guest molecules) and the guest's translational and rotational restraints to the standard concentration, respectively.
+The translational and rotational degrees of freedom of the guest molecule is defined in spherical coordinates
+:math:`(r,\theta,\phi)` and Euler angles :math:`(\alpha,\beta,\gamma)`, respectively. Unlike the work for releasing
+conformational restraints, the work for releasing the guest's restraints to the standard concentration can be evaluated
+analytically:
+
+.. math ::
+   \begin{eqnarray}
+      W_\text{release-std} & = & RT\,\text{ln}\left( \frac{C^{\circ}}{8\pi^2} \right) \\
+         & + & RT\,\text{ln}\left(\int^{\infty}_{0}\int^{\pi}_{0}\int^{2\pi}_{0} e^{-\beta U(r,\theta,\phi)} r^2 \text{sin}(\theta) \, d\phi d\theta dr \right) \\
+         & + & RT\,\text{ln}\left(\int^{2\pi}_{0}\int^{\pi}_{0}\int^{2\pi}_{0} e^{-\beta U(\alpha,\beta,\gamma)} \text{sin}(\theta) \, d\alpha d\beta d\gamma \right)
+   \end{eqnarray}
+   :label: eq:release
+
+For other terms in equation :eq:`eq:bind`, the calculation is broken up into a series of independent windows. In the
+attach and release phase, the windows represents increasing and decreasing force constant, respectively. In the pull
+phase, the windows represents the discretization of the guest path from bound to unbound. The work for each phase is
+obtained using `thermodynamic integration` (TI).
+
+.. math ::
+   \begin{eqnarray}
+      W_\text{attach,release} & = & \int_{0}^{1} \left<F\right>_{\lambda} d\lambda \\
+      W_\text{pull} & = & \int_{A}^{B} \left<F\right>_{r} dr
+   \end{eqnarray}
+   :label: eq:TI
+
+*pAPRika* estimates the standard error of the mean (SEM) with block data analysis.
+
+Binding enthalpy
+================
+
+The binding enthalpy is the difference the partial molar enthalpies of the bound complex and the separated molecules.
+This can be estimated in simulations much easier than the binding free-energy if we use the `direct` method. In the
+`direct` method, the binding enthalpy is obtained from the difference in the mean potential energy of the bound and
+unbound complex.
+
+.. math ::
+   \Delta H_\text{bind} = \left<U_\text{bound} \right> - \left<U_\text{unbound} \right>
+   :label: eq:enthalpy
+
+In the context of APR simulations, the mean potential energy is estimated from the first and last window. These windows,
+however, will need to be run for much longer to get convergence. Another way to estimate the binding enthalpy is with the
```

Comment:
...to get convergence  could maybe be expanded. Like: ...until the estimate of the mean potential energy is more precise. Or until fluctuations die down. Or something about the uncertainty.

Reply:
fixed